test/controllers/reports_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require "test_helper"
+
+class ReportsControllerTest < ActionController::TestCase
+  test "should get index" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.